src/java/org/apache/cassandra/locator/TokenMetadata.java

```diff
@@ -656,6 +662,23 @@ public TokenMetadata cloneOnlyTokenMap()
         }
     }
 
+    private TokenMetadata cloneOnlyTokenMapWithRingVersion()
```

Comment:
Is there a reason why we wouldn't always want to preserve a ring version? We could then just add `ringVersion` to the map above.